engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -126,6 +126,44 @@ FlutterSemanticsFlag SemanticsFlagsToInt(const SemanticsFlags& flags) {
   return static_cast<FlutterSemanticsFlag>(result);
 }
 
+const FlutterSemanticsFlags convertToFlutterSemanticsFlags(
```

Comment:
It's fine to return const pointers and references, but in C++ it's unconventional to return value objects as `const`. If the caller wants to prevent modifications to the returned object it should mark the variable as const at the call site:
```cc
const FlutterSemanticsFlags flags = ConvertToFlutterSemanticsFlags(source);
```